key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -691,8 +710,7 @@ option
 option_list
 :
   LPAREN
-    ( (option (COMMA option)*)
-      | option_expr)
+    (option_expr (COMMA option_expr)*)
```

Comment:
An `option` (`integer:on`) is derivable directly or via `option_expr`. 

This resolves the ambiguity by saying that a list of `option_expr` is allowed, therefore this covers the old behavior of:

```
\rules(programRules:Java, runtimeExceptions:allow) {
```


A list of `option_expr` is treated as a conjunction.

Reply:
You could probably also change `(option (COMMA option)*)` to ` (option (COMMA option)+)` to obtain the desired effect, right?

Reply:
No. You still need to lookbehind the `option` to recognize the if there is a `COMMA`.

Reply:
really? I'd say this resolves ambiguity.